ui/src/components/Ora2PgConfig.vue

```diff
@@ -111,7 +117,7 @@
                 class="mdl-textfield__input" />
               <!-- Checkbox to control whether the parameter should be commented out in
             the ora2pg.conf file at runtime -->
-              <span style="float: right" v-if="item.type !== 'internal'">
+              <span style="float: right" v-if="item.type !== 'internal' && item.type !== 'timestamp'">
```

Comment:
Does timestamp need its own item.type. Could we use internal instead?

Reply:
It is stored in milliseconds but I display it in the UI as a formatted date (which maybe we don't need to do). But that was why it got its own type